cli/args/flags.rs

```diff
@@ -2317,10 +2323,12 @@ fn install_subcommand() -> Command {
 
 Add dependencies to the local project's configuration (<p(245)>deno.json / package.json</>) and installs them 
 in the package cache. If no dependency is specified, installs all dependencies listed in the config file.
+If the <p(245)>--entrypoint</> flag is passed, installs the dependencies of the specified entrypoint(s).
 
   <p(245)>deno install</>
   <p(245)>deno install @std/bytes</>
   <p(245)>deno install npm:chalk</>
+  <p(245)>deno install --entrypoint entry1.ts entry2.ts</>
```

Comment:
We're thinking: `--entrypoint=entrypoint1.ts,entrypoint2.ts`?

Reply:
So I originally did that, but I changed my mind because:
1. The comma separation makes it really noisy combined with the fact that they're paths
2. It doesn't play nice with globbing. I expect people would want to do something like `deno install --entrypoint=src/**/*.ts` which doesn't work if you require the equals and comma delimiter.

Reply:
Yeah those are good points.